### ISSUE TITLE
Ability to filter to windows-curated rulesets

### ISSUE THREAD

**Issue by Miaku** [microsoft/sarif-visualstudio-extension#190]
We need an ability down to the windows-only rulesets in editor, so customers aren't overwhelmed with tons of rulesets when they're trying to pick a ruleset.

**Comment by HenokAddis** [comment]
I don't believe this is a SARIF issue, this could be a WAVE-related issue. This could also be related to the telemetry-interface and identifying what org a user is in so we show relevant rulesets to their organization. I bring this up because *PROJ users won't have WAVE even if they're in the OS repo, and people outside of Windows will be using SARIF.

**Comment by HenokAddis** [comment]
We decided since this is a Windows-specific ask we should get this tracked in AzDO

**Comment by HenokAddis** [comment]
[Bug 28913801](https://microsoft.visualstudio.com/OS/_workitems/edit/28913801): All rules in ruleset.json need clear descriptions on both OSGWiki and visible in the IDE; Need clear way to visualize TVS and MinBar rulesets (if possible)

**Comment by mpande98** [comment]
Is TVS ruleset always applied? If it is, no reason to have it in the ruleset.
